samples/remote_attestation_eeid/README.md

```diff
@@ -0,0 +1,233 @@
+# The Remote Attestation Sample
+
+This sample demonstrates how to do remote attestation between two enclaves and establish a secure communication channel for exchanging messages between them.
+
+It has the following properties:
+
+- Written in C++
+- Demonstrates an implementation of remote attestation
+- Use of mbedTLS within the enclave
+- Use Asymmetric / Public-Key Encryption to establish secure communications between two attesting enclaves
+- Enclave APIs used:
+  - oe_get_report
+  - oe_verify_report,
+  - oe_is_within_enclave
+
+**Note: Currently this sample only works on SGX-FLC systems.** The underlying SGX library support for end-to-end remote attestation is available only on SGX-FLC system. There is no plan to back port those libraries to either SGX1 system or software emulator.
```

Comment:
Is this SGX-specific in some way or should the same sample apply to OP-TEE etc?

Reply:
This is a copy of `samples/remote_attetsation/README.md`, I don't know whether this is still true.